### ISSUE TITLE
TailScale wont start after update from 1.6.0 to 1.8.3 on Synology NAS with AMD CPU

### ISSUE THREAD

**Issue by Markmaster** [tailscale/tailscale-synology#57]
**Description**
After an update from 1.6.0 to 1.8.3 I lost TailScale connection to my NAS. The NAS it was installed not respond to pings from any device inside or outside of my network. I try to restart NAS, but this wasn't helpful. So I try to downgrade to 1.6.0 version but this is unable to do. So I again initiate the setup process and after I have logged in to TailScale again I got the connection to my NAS.
I got three NAS: DS416 - successfully updated and running, DS918+ - successfully update and running, and only DS1621+ has this issue.
**Steps to reproduce**
```
Manually update Synology NAS on AMD CPU, from TailScale 1.6.0 to 1.8.3.
```

**Expected behavior**
I expected that after I update from 1.6.0 as always TailScale will up and running.

**Synology NAS model**
DS1621+

**Comment by apenwarr** [comment]
Decision from eng meeting: our iptables settings never quite worked on every version of Synology, and newer OS versions will stop it working at all. But some people were depending on the borderline behaviour, so let's downgrade this limitation to a warning.

cc: @danderson to correct me if I'm misremembering.

**Comment by DentonGentry** [comment]
Will be resolved when we get https://github.com/tailscale/tailscale/pull/1941 into a 1.8.x release, soon.

**Comment by bradfitz** [comment]
tailscale/tailscale#1941 is unrelated and I still don't think should go in.

There are two main use cases I think we should care about for Synology:

1. I'm not at home and want to get to my File Station web UI.
2. My synology is my only "server" at home, and I want it to be a subnet router (netstack/hybrid mode) so I can get to my weirdo non-Tailscale-able devices when not at home.

Accepting routes is not necessary for either of those use cases.

Reopening this issue.

**Comment by Markmaster** [comment]
@bradfitz Hi. When I'm not at home and i need FileStation Web UI I'll go to QuickConnect. I don't need a direct IP or "white" IP to do that. While I'm not at home I want my file share will appear in my macOS Finder window, to get to my files, or I want my Plex servers from different locations create one big library for me. 

@apenwarr What is "borderline behavior"?!

**Comment by bradfitz** [comment]
Filed https://github.com/tailscale/tailscale/issues/1995 for overall tracking bug on Synology use cases.

**Comment by Markmaster** [comment]
> @Markmaster, what's the IP address of one of your nodes? I want to look at the logs for the machine that's failing to start up.

But if not in my Network how can you do this? Is that secure?

**Comment by Markmaster** [comment]
> @Markmaster, what's the IP address of one of your nodes? I want to look at the logs for the machine that's failing to start up.

@bradfitz But if not in my Network how can you do this? Is that secure?

**Comment by DentonGentry** [comment]
The Tailscale client sends telemetry when it runs into errors. That telemetry is indexed by the tailscale IP address. We cannot connect to IP addresses inside a Tailnet, we just need something to be able to look up whether it is reporting errors in the telemetry.

We'd like to go back to the original problem reported in this issue two weeks ago, after updating to 1.8.3, and see what specific errors it was reporting.

**Comment by Markmaster** [comment]
> The Tailscale client sends telemetry when it runs into errors. That telemetry is indexed by the tailscale IP address. We cannot connect to IP addresses inside a Tailnet, we just need something to be able to look up whether it is reporting errors in the telemetry.
> 
> We'd like to go back to the original problem reported in this issue two weeks ago, after updating to 1.8.3, and see what specific errors it was reporting.

@DentonGentry @bradfitz 100.93.181.24

**Comment by bradfitz** [comment]
Looked at logs. Looks like https://github.com/tailscale/tailscale/pull/1997 will indeed fix it, when 1.8.6 is out.

```
2021-05-28 23:10:54.794080818 +0300 +0300: router: failed to delete legacy rule, continuing anyway: checking for [-m comment --comment tailscale -i tailscale0 -j ACCEPT] in filter/FORWARD:
 running [/sbin/iptables -t filter -C FORWARD -m comment --comment tailscale -i tailscale0 -j ACCEPT --wait]: exit status 2: iptables v1.6.0: Couldn't load match `comment':No such file or 
directory
```
